content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+Documentation
+#############
+
+:menu_order: 007
+
+.. contents::
+
+
+General
+=======
+
+**What is documented?**
+
+**For the Chevah project members:**
+
+* Packages, modules, classes and functions using docstrings. 
+
+* Documentation changes are linked to a certain version of SFTPPlus.  When 
+  changes are made, these are anchored to a specific release version.
+
+**For the customers:**
+
+* These can be pre-sales, trial customers and existing customers.
+
+* There are two main types of documentation - technical documentation and the
```

Comment:
I would say that we have 3 types:

* low level technical information
* high level user guides
* marketing